modules/core/src/lib/layer-manager.js

```diff
@@ -177,9 +177,9 @@ export default class LayerManager {
   /* eslint-enable complexity, max-statements */
 
   // Supply a new layer list, initiating sublayer generation and layer matching
-  setLayers(newLayers) {
+  setLayers(newLayers, force) {
```

Comment:
Nit: Since you are calling this without a second argument in places, maybe add a default value `= false` to call this out?